tests/contracts/rlp/test_plasma_core.py

```diff
@@ -32,18 +32,6 @@ def test_get_output(plasma_core_test):
     assert plasma_core_test.getOutput(tx.encoded, 1) == [null, null, 0]
 
 
-def test_decode_mallability(testlang, plasma_core_test):
```

Comment:
Why removing this test? It checks if one can add multiple IFEs using the same correct transaction.